### DISCUSSION TITLE
Deployment all of a sudden down because of Serverless Function timeout

### DISCUSSION THREAD

**Discussion by kjarrith** [vercel/community#2670] [Help] [answered]
### Summary

What the title says. 

Everything was working nicely yesterday evening over at www.leyndo.is

But then I got a message this morning from one of our users that the whole page was down. No new deployments since then. The only external services we're using are Supabase, GA and Vercel Analytics.

I have no Serverless Functions of my own setup. Nothing helpful in the logs unless the fact that the primary GET request to "/" is failing.

I already tried redeploying but that did not help. Everything works nicely locally. 

What could be happening?

<img width="1135" alt="Screenshot 2023-05-31 at 08 52 11" src="https://github.com/vercel/community/assets/5184039/06b369b3-ceda-4cdb-8062-21cc61856bc9">

### Example

www.leyndo.is

### Steps to Reproduce

This happens on every load.

<img width="872" alt="Screenshot 2023-05-31 at 08 49 04" src="https://github.com/vercel/community/assets/5184039/65766b94-2658-404a-893d-b2be8d5c5595">

**Comment by kjarrith** [comment]
Update: It's working now. 

Very unfortunate to not now the reason for why it went down, or up

**Comment by amyegan** [comment, marked as answer]
Glad to see it's working now. A `504` [FUNCTION_INVOCATION_TIMEOUT](https://vercel.com/docs/errors/application-error-codes#function_invocation_timeout) means the Serverless Function reached the [Serverless Function Execution Timeout](https://vercel.com/docs/concepts/limits/overview#serverless-function-execution-timeout) limit. Unhandled exceptions or delays in returning requests from external services are the most common causes, so those are things to look out for if you run into this error in the future.
